draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -983,6 +1003,14 @@ when presenting to a downstream party.
 In some scenarios this behavior could be desirable,
 but if it is not, Issuers need to support and Verifiers need to enforce Key Binding.
 
+## Integrity of Presentation
+
+In a Presentation, the Issuer-signed JWT is integrity-protected by the Issuer's
+signature, and the Disclosures are integrity-protected by the digests included
+in the Issuer-signed JWT. However, without enforcing a Key Binding JWT as
+defined in (#kb-jwt), the integrity of the set of Disclosures the
+Holder selected to send is not ensured.
```

Comment:
```suggestion
Holder disclosed is not ensured.
```

Reply:
I might rework the phrasing of this section,

The specific set of disclosures presented MAY be integrity protected through the use of a hash or other means, etc... it reads a bit awkward as is